tests/functionality/scopflow/selfcheck.cpp

```diff
@@ -191,13 +253,14 @@ struct ScopflowFunctionalityTests
   void run_test_case(Params &params) override {
     PetscErrorCode ierr;
     SCOPFLOW scopflow;
+    char pbuf[PETSC_MAX_PATH_LEN];
     int rank;
+    auto err = MPI_Comm_rank(comm, &rank);
+    if (err)
+      throw ExaGOError("Error getting MPI rank number");
 
-    MPI_Comm_rank(PETSC_COMM_WORLD, &rank);
-
-    if (rank == 0) {
+    if (rank == logging_rank)
```

Comment:
Just use 0... Maybe just define value in header file if you really want to configure this.